packages/eslint-plugin/tests/docs-eslint-output-snapshots/no-deprecated.shot

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Validating rule docs no-deprecated.mdx code examples ESLint output 1`] = `
+"Incorrect
+
+/** @deprecated Use @see {@link apiV2} instead. */
+declare function apiV1(): Promise<string>;
+
+declare function apiV2(): Promise<string>;
+
+await apiV1();
+      ~~~~~ \`apiV1\` is deprecated. Use
```

Comment:
this doesn't seem right?

Reply:
Ah, I'm using the tags wrong apparently. `@see` can't be nested inside `@deprecated`. They're two different tags from `symbol.getJsdocTags()`.

![Screenshot of hovering over apiV1 from the linked playground, showing @deprecated and @see on a new line each](https://github.com/user-attachments/assets/5a5b5748-951c-437e-ab27-73e990e804d5)

I'll just remove the extra tags.